pyhomematic/devicetypes/actors.py

```diff
@@ -594,15 +594,27 @@ def __init__(self, device_description, proxy, resolveparamsets=False):
         # init metadata
         self.SENSORNODE.update({"DOOR_STATE": [1]})
 
-    def move_up(self):
+    def is_closed(self):
```

Comment:
Shouldn't the channel be passed here?

Reply:
The (new) code calling it doesn't set the channel; it is hard-coded to 1 anyway. I can try also passing the channel here.

Regardless, this method (is_closed) might need to be changed, according to @pvizeli's comment in the home assistant PR

Reply:
Yeah, he's talking about a cached value. But we don't really have that on a global scope. Have a look at [this](https://github.com/danielperna84/pyhomematic/search?q=cached&unscoped_q=cached). There's a light that has implemented some caching for itself. Maybe that can be reused for your purpose.

Reply:
I've changed the method to just resolve a map "state -> closed?". Now everything seems to work (and I'm now using the cached value like e.g. "position" is doing").